src/config/authorsConfig.ts

```diff
@@ -15,6 +15,18 @@ const authorsConfig: AuthorConfig[] = [
       "What are sighashes?",
     ],
   },
+  {
+    name: "CoreDev Bot",
+    title: "Bitcoin Core Seer",
+    introduction: "Hi, I'm a bitcoin core expert. I'm only trained on Bitcoin Core sources so I don't know anything about the greater ecosystem but if you wnat to learn about Bitcoin Core development ask away.",
```

Comment:
s/wnat/want

Reply:
Fixed in #41. Thanks for the catch @brunoerg